llama_index/vector_stores/types.py

```diff
@@ -56,22 +56,34 @@ class VectorStoreQueryMode(str, Enum):
     LINEAR_REGRESSION = "linear_regression"
 
 
+@dataclass
+class ExactMatchFilter:
```

Comment:
so this and metadata filters are exposed to the user - can we make sure we add docstrings and document? in references at the very least and ideally in the vector store guide

Reply:
Added docstring, guide, and reference